lib/location/location_utils.c

```diff
@@ -109,3 +109,110 @@ void location_utils_systime_to_location_datetime(struct location_datetime *datet
 	datetime->second = ltm.tm_sec;
 	datetime->ms = tp.tv_nsec / 1000000;
 }
+
+static K_MUTEX_DEFINE(list_mtx);
+
+/**@brief List element for event handler list. */
+struct event_handler {
+	sys_snode_t node;
+	location_event_handler_t handler;
+};
+
+static sys_slist_t handler_list;
+
+/**
+ * @brief Find the handler from the event handler list.
+ *
+ * @return The node or NULL if not found and its previous node in @p prev_out.
+ */
+static struct event_handler *location_utils_event_handler_find(
+	struct event_handler **prev_out,
+	location_event_handler_t handler)
+{
+	struct event_handler *prev = NULL, *curr;
```

Comment:
Linux kernel coding style advices against this:

_To this end, use just one data declaration per line (no commas for multiple data declarations)._

Check `location_utils_event_handler_remove()` and `location_utils_event_dispatch()` as well.

Reply:
I'll blame copy-paste 😄